README.md

```diff
@@ -1,19 +1,31 @@
-Minecraft Console Client
-========================
+<div align="center">
+
+<img src="https://i.pics.rs/LLDhE.png" alt="Logo"/>
+
+# Minecraft Console Client (MCC)
+
+[Documentation](https://mccteam.github.io/docs/) | [Download](#download-🔽) | [Installation](https://mccteam.github.io/docs/guide/installation.html) | [Configuration](https://mccteam.github.io/docs/guide/configuration.html) | [Usage](https://mccteam.github.io/docs/guide/usage.html)
```

Comment:
The Download link does not seems to jump to the appropriate section, likely because of the 🔽 emoticon 🤔

Reply:
I'll remove it then.